### ISSUE TITLE
dotnet fsi.exe doesn't resolve some symbols

### ISSUE THREAD

**Issue by kontomondo** [Microsoft/visualfsharp#4717]
Apologies if fsi.exe isn't supposed to be used yet, I've been running it since the .NET Core 2.1 preview1 & preview2. Trying to say use `File.ReadAllLines` after doing `open System.IO` results in an error 

#### Repro steps
Run `dotnet fsi.exe`, then:
```
> open System.IO;;
> File.ReadAllLines;;
error FS0039: The value, namespace, type or module 'File' is not defined. Maybe you want one of the following:
   FileMode
   FileShare
   FileAccess
   FileStream
   FileOptions
```

#### Expected behavior
```
> open System.IO;;
> File.ReadAllLines;;
val it : arg00:string -> string []
```

#### Known workarounds

Use fsharpi from Mono

#### Related information 

* Linux
* .NET Core 2.1 preview2

**Comment by dsyme** [comment]
cc @KevinRansom @cartermp 

``dotnet fsi.exe`` is not yet fully implemented, seee #2407

**Comment by dsyme** [comment]
Closing as duplicate of #2407
